INSTALL.rst

```diff
@@ -281,7 +281,7 @@ openPMD API
 - environment:* (assumes install from source in ``$HOME/lib/openPMD-api``)
 
   - ``export CMAKE_PREFIX_PATH="$HOME/lib/openPMD-api:$CMAKE_PREFIX_PATH"``
-  - ``export LD_LIBRARY_PATH="$HOME/lib/openPMD-api/lib:$LD_LIBRARY_PATH"``
+  - ``export LD_LIBRARY_PATH="$HOME/lib/openPMD-api/lib64:$LD_LIBRARY_PATH"``
```

Comment:
It is platform-specific whether the shared library is installed under `lib` or `lib64`. 
```suggestion
  - ``export LD_LIBRARY_PATH="$HOME/lib/openPMD-api/lib64:$HOME/lib/openPMD-api/lib:$LD_LIBRARY_PATH"``
```

Reply:
Great suggestion! Will include.